src/bitcoin-cli.cpp

```diff
@@ -991,6 +1057,11 @@ static int CommandLineRPC(int argc, char *argv[])
                 if (gArgs.IsArgSet("-getinfo") && !gArgs.IsArgSet("-rpcwallet")) {
                     GetWalletBalances(result); // fetch multiwallet balances and append to result
                 }
+
+                if (gArgs.IsArgSet("-getinfo")) {
+                    ParseGetInfoResult(result);
+                }
+
```

Comment:
nit, maybe combine the two -getinfo checks, feel free to ignore
```diff
             if (error.isNull()) {
-                if (gArgs.IsArgSet("-getinfo") && !gArgs.IsArgSet("-rpcwallet")) {
-                    GetWalletBalances(result); // fetch multiwallet balances and append to result
-                }
-
                 if (gArgs.IsArgSet("-getinfo")) {
+                    if (!gArgs.IsArgSet("-rpcwallet")) {
+                        GetWalletBalances(result); // fetch multiwallet balances and append to result
+                    }
                     ParseGetInfoResult(result);
                 }
```

Reply:
Followed your suggestion and combined the checks